### PR TITLE
Make flutter run_mojo imply flutter build

### PR DESCRIPTION
**Pull request by abarth**
This makes run_mojo more similar to flutter start.

### CODE REVIEW

**Comment by abarth** [comment]
@krisgiesing

File: packages/flutter_tools/lib/src/commands/build.dart [hunk 1]
```diff
@@ -109,8 +110,8 @@ ArchiveFile _createSnapshotFile(String snapshotPath) {
 const String _kDefaultAssetBase = 'packages/material_design_icons/icons';
 const String _kDefaultMainPath = 'lib/main.dart';
 const String _kDefaultManifestPath = 'flutter.yaml';
-const String _kDefaultOutputPath = 'app.flx';
-const String _kDefaultSnapshotPath = 'snapshot_blob.bin';
+const String _kDefaultOutputPath = 'build/app.flx';
+const String _kDefaultSnapshotPath = 'build/snapshot_blob.bin';
```
Comment (krisgiesing) [inline packages/flutter_tools/lib/src/commands/build.dart]: git ignore this directory?

File: packages/flutter_tools/lib/src/commands/run_mojo.dart [hunk 2]
```diff
@@ -25,7 +29,11 @@ class RunMojoCommand extends FlutterCommand {
     argParser.addFlag('mojo-debug', negatable: false, help: 'Use Debug build of mojo');
     argParser.addFlag('mojo-release', negatable: false, help: 'Use Release build of mojo (default)');
 
-    argParser.addOption('app', defaultsTo: 'app.flx');
+    argParser.addOption('target',
+        defaultsTo: '',
```
Comment (krisgiesing) [inline packages/flutter_tools/lib/src/commands/run_mojo.dart]: Why no defaults?

File: packages/flutter_tools/lib/src/commands/build.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter_tools/lib/src/commands/build.dart]: We already do.

File: packages/flutter_tools/lib/src/commands/run_mojo.dart (hunk 2 above)
Comment (abarth) [inline packages/flutter_tools/lib/src/commands/run_mojo.dart]: Defaulting to the empty string is effectively defaulting to the current working directory, which means defaulting to 'lib/main.dart'.

**Comment by krisgiesing** [comment]
LGTM

**Pull request merged**